Library/Homebrew/os/mac.rb

```diff
@@ -120,6 +120,24 @@ def clang_build_version
         end
     end
 
+    def clang_omp_version
+      @clang_omp_version ||=
+        if (path = locate("clang-omp"))
+          %x{#{path} --version}[/clang version (\d\.\d)/, 1]
+        end
+    end
+
+    # Create a pseudo build version number based on Apple Clang build version and
+    # its LLVM version. This is hack to make clang-omp's version comparable with
+    # Apple Clang build version. Please update this map with a proper build number
+    # when clang-omp gets update.
+    def clang_omp_build_version
+      @clang_omp_build_version ||=
+        case clang_omp_version
+        when "3.5" then 600
+        end
+    end
```

Comment:
I have no strong complains to this approach. I also haven't yet tested the patch.